package.json

```diff
@@ -5,11 +5,16 @@
   "author": "Karin Strouhalů <karin.strouhalu@nic.cz>",
   "license": "GPL-3.0",
   "main": "dist/django-forms-vuetify.common.js",
+  "homepage": "https://github.com/CZ-NIC/django-forms-vuetify",
+  "repository": {"type": "git", "url": "https://github.com/CZ-NIC/django-forms-vuetify"},
+  "keywords": ["django-forms", "vuetify", "dynamic form", "dynamic vuetify from"],
+  "bugs": {"url": "https://github.com/CZ-NIC/django-forms-vuetify/issues"},
   "scripts": {
     "build": "vue-cli-service build --target lib --name 'django-forms-vuetify' src/main.ts",
     "check": "vue-cli-service lint --max-errors 0 --max-warnings 0 tests src",
     "test": "vue-cli-service test:unit",
-    "watch": "NODE_ENV=production vue-cli-service build --watch --mode development --target lib --name 'django-forms-vuetify' src/main.ts"
+    "watch": "NODE_ENV=production vue-cli-service build --watch --mode development --target lib --name 'django-forms-vuetify' src/main.ts",
```

Comment:
Maybe this is from some previous branch but why are we setting `NODE_ENV` to `production` and at the same time set `--mode development`?

Reply:
Yes, it is from previous branch. It is small hack, because without `NODE_ENV=production` I had problems with successful building in watch mode in init version (this was only one workaround which I found). But now it seems that work without it also, so I am removing it.